pages/register.tsx

```diff
@@ -47,25 +73,38 @@ const Home: NextPage = () => {
             <FormControl color="white.700">
               <FormControl isRequired mt={5}>
                 <FormLabel htmlFor="imageFile">Picture</FormLabel>
-                <Input
-                  variant="outline"
-                  id="imageFile"
-                  type="file"
-                  accept={'image/*'}
-                  isRequired={true}
-                  placeholder="Your nengajo"
-                  name="profilePicture"
-                  onChange={handleImageChange}
-                />
+
+                <form onSubmit={sendFileToIPFS}>
+                  <Input
+                    variant="outline"
+                    id="imageFile"
+                    type="file"
+                    accept={'image/*'}
+                    isRequired={true}
+                    placeholder="Your nengajo"
+                    name="profilePicture"
+                    onChange={handleImageChange}
+                  />
+                  {isSuccess ?
```

Comment:
I think we don't need the isSuccess state variable.

Instead of checking for `isSuccess`, check for `imageUri !== ''`.

Then you can delete everything related to `isSuccess` and `setIsSuccess`.